tests/jballerina-unit-test/src/test/resources/test-src/expressions/stamp/json-stamp-expr-test.bal

```diff
@@ -175,3 +189,14 @@ function stampNullJSONToArrayNegative() returns StringArray|error {
 
     return s;
 }
+
+function assert(anydata actual, anydata expected) {
+    if (expected != actual) {
```

Comment:
Can check `expected == actual` and return early, and get rid of the indentation.